spec/controllers/catalog_controller_spec.rb

```diff
@@ -99,6 +99,21 @@
       post :create, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
       expect(response.body).to include(druid_no_pre)
     end
+    context 'can take two forms of druid input' do
+      let(:druid) { 'druid:jj925bx9565' }
+      let(:druid_no_pre) { 'jj925bx9565' }
+
+      it 'druid:jj925bx9565' do
+        post :create, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
+        expect(response.body).to include(druid_no_pre)
```

Comment:
I'm a little unclear about this expectation: the response body equals `druid` and includes `druid_no_pre`. Would a more rigorous expectation be that the body equals `druid`? I understand that the body includes the String `jj925bx9565`, but `druid_no_pre` makes me think it's a druid, not a String, and then I get hung up on the weird thought that a druid includes a druid. Or do we need this expectation at all? Is it enough to expect a `:created` status?

Reply:
So, i'm just trying to show that you can give the `POST` or `PATCH` request either `druid:jj925bx9565` or `jj925bx9565` and the request will go through, and not throw an error. The catalog controller code will take either input do a `.split(':', 2)` and return `jj925bx9565` in both cases. So its just like oh yeah send the create params w/ `druid:jj925bx9565`, but we'll end up w/ `jj925bx9565` bc of internal code.

I could also add a test thats like 
`expect(response.body).no_to include(druid)`


I can remove this test and just keep the status code. If you think that would be more clear.

Reply:
The additional `expect(response.body).not_to include(druid)`makes it clearer to my brain.

Reply:
and here, in the expectations, using the names `bare_druid` and `prefixed_druid` would help make this clearer, too.  Also, I agree with @tallenaz

Reply:
are we expecting it NOT to include prefixed_druid  as well as  including  bare_druid?    You could write expectations to make this abundantly clear.